Tests/WorkspaceTests/InitTests.swift

```diff
@@ -23,10 +23,11 @@ class InitTests: XCTestCase {
         mktmpdir { tmpPath in
```

Comment:
We should add a test for the new functionality.

Reply:
Done 👍